### DISCUSSION TITLE
Accessing player props markets via the API

### DISCUSSION THREAD

**Discussion by codelancers-sd** [cloudbet/docs#87] [Q&A]
Hello,

I have been using the cloudbet API for a short time and have managed to get/place bets quite comfortably. However, I am struggling to access specific markets. For example, in baseball, I can see markets for player_props listed on the webpage. (Those markets are also available under "Specials", as far as I know for a certain match). However, there does not seem a way to access those markets via the api. For example, the api offers these for baseball:

1. baseball.moneyline
2. baseball.total
3. Etc... (more here: https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7#file-markets-json-L842)

But I cannot find a way to access things like: "Player A to have more than 5.5 strikeouts". Those kind of markets are also not available under specials.outright. Is there a way I can get to those? Any help would be much appreciated!

**Comment by kgravenreuth** [comment]
Hi @codelancers-sd ,
The full list of markets available on the Cloudbet API is available at https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7#file-markets-json-L842 as you already found. At the moment, this may not include all the markets available via the Cloudbet website. 

We'll let you know if or when we update the list of available markets on the Cloudbet API. Please let me know if you have any other questions.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by codelancers-sd** [reply]
I see thanks. I would greatly appreciate if this were made available!
